src/Servers.jl

```diff
@@ -324,6 +324,17 @@ function listen(f, listener::Listener; kw...)
     return server
 end
 
+# compat for `Threads.@spawn :interactive expr`
+@static if hasmethod(getfield(Threads, Symbol("@spawn")), Tuple{LineNumberNode, Module, Symbol, Expr})
+    macro _spawn_interactive(ex)
+        esc(:(Threads.@spawn :interactive $ex))
+    end
+else
+    macro _spawn_interactive(ex)
+        esc(:(Threads.@spawn $ex))
+    end
```

Comment:
Should this be `@async` to keep it more similar to the behavior pre-1.9?

Reply:
Yeah, that's probably a good idea.